test/createTable.test.ts

```diff
@@ -26,18 +27,45 @@ describe("create method", function () {
 
   test("Create table works", async function () {
     fetch.mockResponseOnce(FetchCreateDryRunSuccess);
-    fetch.mockResponseOnce(FetchCreateTableOnTablelandSuccess);
+    fetch.mockResponseOnce(FetchReceiptExists);
 
     const txReceipt = await connection.create("id int primary key, val text");
     await expect(txReceipt.tableId._hex).toEqual("0x015");
   });
 
   test("Create table throws if dryrun fails", async function () {
     fetch.mockResponseOnce(FetchCreateDryRunError);
-    fetch.mockResponseOnce(FetchCreateTableOnTablelandSuccess);
+    fetch.mockResponseOnce(FetchReceiptExists);
 
     await expect(async function () {
       await connection.create("id int primary key, val text", "123test");
     }).rejects.toThrow("TEST ERROR: invalid sql near 123");
   });
+
+  test("Create table waits to return until after materialization", async function () {
+    fetch.mockResponseOnce(FetchCreateDryRunSuccess);
+    fetch.mockResponseOnce(FetchReceiptNone);
+    fetch.mockResponseOnce(FetchReceiptExists);
+
+    const txReceipt = await connection.create("id int primary key, val text");
+    await expect(txReceipt.tableId._hex).toEqual("0x015");
+  });
+
+  test("Create table options enable not waiting to return until after materialization", async function () {
+    fetch.mockResponseOnce(FetchCreateDryRunSuccess);
+
+    const txReceipt = await connection.create("id int primary key, val text", { skipConfirm: true });
+    await expect(txReceipt.tableId._hex).toEqual("0x015");
+  });
+
+  test("Create table options enable setting timeout for materialization", async function () {
+    fetch.mockResponseOnce(FetchCreateDryRunSuccess);
+    fetch.mockResponseOnce(FetchReceiptNone);
+    fetch.mockResponseOnce(FetchReceiptNone);
+    fetch.mockResponseOnce(FetchReceiptNone);
+
+    await expect(async function () {
+      await connection.create("id int primary key, val text", { timeout: 2000 /* 2 seconds */ })
+    }).rejects.toThrow(/timeout exceeded: could not get transaction receipt:/);
```

Comment:
What is this magical slash thing?

Reply:
good old fashion literals, plus got to use the character string :/) !!